<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpWindow</class>
 <widget class="QMainWindow" name="SignUpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign up</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/src.qrc">
    <normaloff>:/images/login.png</normaloff>:/images/login.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.551, y1:0.835, x2:0.517236, y2:0.051, stop:0 rgba(0, 255, 0, 223), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <widget class="QGroupBox" name="signUpForm">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>371</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.511, y1:1, x2:0.527831, y2:0, stop:1 rgba(68, 255, 51, 0));
border-color: rgb(0, 170, 0);
border-size: 2px;
font: 75 14pt &quot;Calibri&quot;;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 134, 0);
font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="title">
     <string>Please sign up</string>
    </property>
    <widget class="QLineEdit" name="nameLine">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>73</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">borde-sizer: 30px;
borde-color: red;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">borde-sizer: 30px;
borde-color: red;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="signUpButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>300</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#signUpButton{
	color: rgb(0, 170, 0);
	color: rgb(255, 255, 255);
	background-color: rgb(0, 145, 0);
	border-radius: 10px;
}
#signUpButton:hover{
	background-color:#00aa00;
}</string>
     </property>
     <property name="text">
      <string>sign up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backToLoginButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#backToLoginButton{
background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 0, 0, 0));
color: rgb(0, 134, 0);
font-size: 16px;
border:none;
}

#backToLoginButton:hover{
color: #ffffff;
border:none;
}</string>
     </property>
     <property name="text">
      <string>Back to login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="repeatPasswordLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 132, 255);
color: rgb(0, 153, 0);
background-color: qlineargradient(spread:pad, x1:0.511, y1:1, x2:0.510978, y2:0, stop:0 rgba(97, 255, 255, 0), stop:1 rgba(255, 255, 255, 255));
font: 75 24pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Welcome to Heavy Wallet</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/src.qrc"/>
 </resources>
 <connections/>
</ui>
